developerportal/templates/molecules/card-featured.html

```diff
@@ -1,27 +1,54 @@
 {% load static %}
 {% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
-{% image resource.image width-480 as image %}
-{% image resource.card_image width-480 as card_image %}
-{% if block_type == "video" %}
-  {% static "img/placeholders/video.jpg" as fallback_image_url %}
-{% elif block_type == "event" %}
-  {% static "img/placeholders/event.jpg" as fallback_image_url %}
+
+{% if resource.card_image %}
+  {% image resource.card_image width-480 as img %}
 {% else %}
-  {% static "img/placeholders/article.jpg" as fallback_image_url %}
+  {% if resource.image %}
+    {% image resource.image width-480 as img %}
+  {% else %}
+    {% if block_type == "video" %}
+      {% static "img/placeholders/video.jpg" as fallback_url %}
+    {% elif block_type == "event" %}
+      {% static "img/placeholders/event.jpg" as fallback_url %}
+    {% else %}
+      {% static "img/placeholders/article.jpg" as fallback_url %}
+    {% endif %}
+  {% endif %}
 {% endif %}
 
+{% comment %}
+Note that this partial needs wrapping with a div and a section with the appropriate classes
+{% endcomment %}
 {% if external_page %}
-<a href="{{ resource.url }}" class="card-link" data-type="external_page" target="_blank" rel="nofollow noopener">
+<a href="{{ resource.url }}" class="mzp-c-card-block-link" data-type="external_page" target="_blank" rel="nofollow noopener">
 {% else %}
-<a href="{% pageurl resource %}" class="card-link" data-type="{{ resource.resource_type }}">
+<a href="{% pageurl resource %}" class="mzp-c-card-block-link" data-type="{{ resource.resource_type }}">
 {% endif %}
-  <div class="card-featured">
-    <div class="card-featured-image" style="background-image: url('{% firstof card_image.url image.url fallback_image_url %}')"></div>
-    <div class="card-featured-caption">
-      <h5 class="no-underline">
-        {% firstof resource.card_title resource.title %}{% if external_page or resource.is_external %}<span class="no-wrap">&#65279;<span class="icon icon-external">{% include "atoms/icons/external.svg" %}</span></span>{% endif %}
-      </h5>
+  <div class="mzp-c-card-media-wrapper">
+      <div class="lazy-image-container">
+        <img class="mzp-c-card-image" src="{% firstof img.url fallback_url %}" alt="">
+      </div>
+  </div>
```

Comment:
General question: Why do we need two wrapping `div` elements for the one `img`. I guess the CSS framework might be enforcing that.